packages/puppeteer-core/src/common/Input.ts

```diff
@@ -598,6 +598,33 @@ export class Mouse {
     }
   }
 
+  /**
+   * Resets the mouse to the default state. No buttons pressed; position at
+   * (0,0).
+   */
+  async reset(): Promise<void> {
+    const actions = [];
+    if (this.#state.buttons & MouseButtonFlag.Left) {
+      actions.push(this.up({button: MouseButton.Left}));
+    }
+    if (this.#state.buttons & MouseButtonFlag.Middle) {
+      actions.push(this.up({button: MouseButton.Middle}));
+    }
+    if (this.#state.buttons & MouseButtonFlag.Right) {
+      actions.push(this.up({button: MouseButton.Right}));
+    }
+    if (this.#state.buttons & MouseButtonFlag.Forward) {
+      actions.push(this.up({button: MouseButton.Forward}));
+    }
+    if (this.#state.buttons & MouseButtonFlag.Back) {
```

Comment:
want to convert it to an array + iteration? should be easier to read and less chance for copy paste errors.

Reply:
I can alternatively create an array manually.

Reply:
yeah, I meant the former option.

Reply:
i.e.,

```
const flags = [MouseButtonFlag.Left, ...];
for (const button of flags) {
   if (this.#state.buttons & button) {
      actions.push(this.up({button}));
    }
}
```